README.md

```diff
@@ -1,60 +1,85 @@
 CakePHP Sample App on OpenShift
------------------
+===============================
 
-This example will serve an http response of various "machine" stats from the "machine" the CakePHP app is running on to [http://host:8080](http://host:8080).
+This is a quickstart CakePHP application for OpenShift v3 that you can use as a starting point to develop your own application and deploy it on an [OpenShift](https://github.com/openshift/origin) cluster.
+
+If you'd like to install it, follow [these directions](https://github.com/openshift/cakephp-ex/blob/master/README.md#installation).  
 
-### OpenShift setup ###
+The steps in this assume that you have access to an OpenShift deployment; you must have an OpenShift deployment that you have access to in order to deploy this app.
+
+This example will serve an http response of various "machine" stats from the "machine" the CakePHP app is running on to [http://host:8080](http://host:8080).
```

Comment:
When I think about it now I would get rid of this whole sentence and put rather something like we have in Rails-ex.

In order to access the example CakePHP home page, which contains application stats, you have to go into the `app/Views/Layouts/` directory, remove the `default.ctp` and  after that rename `default.ctp.default` into default.ctp`.

Also would move this after the 'If you'd like to install it, follow these directions.' sentence as we have it in rails-ex